File: easy/cpp/0119.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> start{1};
        if(!rowIndex) return start;
        int cnt = 0;
        while(cnt <= rowIndex){
            vector<int> next{1};
            for(int i = 1 ; i < cnt; i++){
                next.push_back(start[i-1] + start[i]);
            }
            next.push_back(1);
            start = next;
            cnt++;
        }
        return start;
    }
};

File: easy/python/0119.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        start = [1]
        if not rowIndex: return start
        while rowIndex:
            next_start = []
            for i in range(len(start) - 1):
                next_start.append(start[i] + start[i+1])
            start = [1] + next_start + [1]
            rowIndex -= 1
        return start
            